examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -1,28 +1,35 @@
 var fs = require('fs');
 var util = require('util');
 var request = require("request");
+var cmdline = require("./cmdline");
 
-var host = process.argv[2] || 'http-api-host';
-var port = 3000;
-var url = 'https://' + host + ':' + port;
-var NUM_REQUEST = 10;   // Number of long-poll requests before unsubscribe
+var NUM_REQUEST = 10; // Number of long-poll requests before unsubscribe.
 var counter = 0;
-var opt = {
-    cert: fs.readFileSync('client.crt'),
-    key: fs.readFileSync('client.key'),
-    ca: fs.readFileSync('bloomberg.crt')
-};
+
+function url()
```

Comment:
I changed it to a function so this code is not not evaluated before the `subscribe()` and `cmdline.parse()` function calls. If it is evaluated before that and you don't have the certificate files around you won't be even able to print usage, because we won't even get there, hence the function. Any other suggestions in presence of this argument?